testsuite/src/test/groovy/noe/tomcat/BindingsTomcat7ConfiguratorIT.groovy

```diff
@@ -1,12 +1,15 @@
 package noe.tomcat
 
+import noe.common.utils.Platform
+import org.junit.Assume
 import org.junit.BeforeClass
 
 
 class BindingsTomcat7ConfiguratorIT extends BindingsTomcatConfiguratorIT {
 
   @BeforeClass
   public static void beforeClass() {
+    Assume.assumeTrue("Tomcat 7 not release for >RHEL7", new Platform().isRHEL7())
```

Comment:
Example of multiple types or limitations. In other places you have RHEL8+

Reply:
Ah, I see! I'll change it